### PR TITLE
ready to code review

### PR DESCRIPTION
**Pull request by xtarzyk**

### CODE REVIEW

File: circles.js
Comment (jpudysz) [inline circles.js:6]: Maybe you can put it into separate function like `prepareUI` or `createGameUI`

File: circles.js
Comment (jpudysz) [inline circles.js:5]: spaces between `{}`

File: circles.js
Comment (jpudysz) [inline circles.js:10]: before `let const` and `instructions` add enter

File: circles.js
Comment (jpudysz) [inline circles.js:14]: bad mutation, why can't you use `index`?

File: circles.js
Comment (jpudysz) [inline circles.js:33]: enter before `const` and instructions

File: circles.js
Comment (jpudysz) [inline circles.js:32]: you're doing query selector each time you invoke `resetColors`, thats a bad practice

File: circles.js
Comment (jpudysz) [inline circles.js:20]: same here, check comment below

File: circles.js
Comment (jpudysz) [inline circles.js:23]: unnecessary `()`

File: circles.js
Comment (jpudysz) [inline circles.js:24]: I don't get this if

File: circles.js
Comment (jpudysz) [inline circles.js:42]:
```
export {
    gameContainer, 
    paintCircle,
    resetColors,
    createCircles
}
```

File: main.js
Comment (jpudysz) [inline main.js:1]: single quote

File: main.js
Comment (jpudysz) [inline main.js:2]: spaces between `{}`

File: package.json
Comment (jpudysz) [inline package.json:13]: styles.css and styles.css.map to gitignore

File: timer.js
Comment (jpudysz) [inline timer.js:1]: singlequote

File: timer.js
Comment (jpudysz) [inline timer.js:34]: spaces between `{}`

File: timer.js
Comment (jpudysz) [inline timer.js:10]: you can use single timer like: `seconds`

File: timer.js
Comment (jpudysz) [inline timer.js:13]:
```
if (seconds / 60 > 0)
```

File: timer.js
Comment (jpudysz) [inline timer.js:20]:
```
if (seconds % 60 >= 0)
```

File: timer.js
Comment (jpudysz) [inline timer.js:26]: please avoid `else`

**Review by jpudysz — CHANGES_REQUESTED** [review]

File: circles.js
Comment (xtarzyk) [reply circles.js:14]: true

File: circles.js
Comment (xtarzyk) [reply circles.js:32]: every time the resetColors is invoke it needs to get current state of all circles divs, otherwise it's not works

**Review by xtarzyk — COMMENTED** [review]
done

**Comment by jpudysz** [comment]
You can't merge it. It will be merged after approve

**Comment by jpudysz** [comment]
And I can't see any fixes. Please git push them

**Pull request merged**
